api/database/factories/ModelFactory.php

```diff
@@ -138,6 +138,16 @@
     ];
 });
 
+$factory->define(App\Models\Image::class, function (\Faker\Generator $faker) {
+    return [
+            'image_id' => $faker->uuid,
+            'public_id' => $faker->unique()->word,
```

Comment:
This faker property is _very_ collision prone, and it doesn't represent a cloudinary `public_id`. Use `$faker->unique()->lexify(????????????????????)` instead